.github/workflows/build-release.yml

```diff
@@ -184,3 +184,39 @@ jobs:
 
           cd ../installation
           ./bin/python3 -m test -uall
+
+  build-android:
+    needs:
+      - verify-input
+
+    # Android binary releases began in Python 3.14.
+    if: |
+       !(
+          startsWith(needs.verify-input.outputs.cpython_release, '3.9.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.10.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.11.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.12.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.13.')
+       )
```

Comment:
Would this work?
```suggestion
    if: ${{ needs.verify-input.outputs.cpython_release >= '3.14' }}
```

Reply:
@mhsmith would [`fromJSON`](https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#fromjson) work? That would produce a float value, but we could then use proper equality operators.

Reply:
Okay thanks, an explicit list isn't so bad, YAML being what it is :)

Reply:
And `3.9 > 3.14` in a float comparison as well.